lib/console_web/controllers/router/device_controller.ex

```diff
@@ -154,6 +154,16 @@ defmodule ConsoleWeb.Router.DeviceController do
         result =
           Ecto.Multi.new()
           |> Ecto.Multi.run(:event, fn _repo, _ ->
+            event = case event["data"]["req"]["body"] do
```

Comment:
https://appsignal.com/helium-systems-inc/sites/5ebed0327478204c2e688644/exceptions/incidents/38?activeFilter=timeline&selectedTime=2021-06-26T00%3A00%3A00.000Z

Reply:
I think event["data"]["req"]["body"] fails ungracefully if "data" obj does not have "req" or "req" obj does not have "body". I know the line below doesn't handle it well either (my fault), but can you make a quick edit for both of them?

Reply:
nvm ignore, i tested on local and its fine. Its not fine in iex -_-